pyscriptjs/tests/integration/support.py

```diff
@@ -197,6 +197,7 @@ def pyscript_run(self, snippet):
           <head>
               <link rel="stylesheet" href="{self.http_server}/build/pyscript.css" />
               <script defer src="{self.http_server}/build/pyscript.js"></script>
+              {config}
```

Comment:
+1 for introducing a way to inject extra stuff in the head, but the name `config` sounds wrong.
I would call it `extra_head`, and I would make it a keyword-only argument, i.e.:
```python
def pyscript_run(self, snippet, *, extra_head=""):
```

Reply:
thanks, renamed!